src/files.ts

```diff
@@ -295,8 +295,10 @@ export async function prepareExtendingBundle(
   let processingFiles: string[] = filterSupportedFiles(files, supportedFiles);
 
   // Exclude files to be ignored based on ignore rules. We assume here, that ignore rules have not been changed.
-  processingFiles = processingFiles.filter(f => !isMatch(f, fileIgnores));
-
+  processingFiles = processingFiles
+    .map(f => resolveBundleFilePath(baseDir, f))
+    .filter(f => !isMatch(resolveBundleFilePath(baseDir, f), fileIgnores));
```

Comment:
This seems weird, the map above maps `processingFiles` with `resolveBundleFilePath`. Why do you need to pass it here again.

Put differently, what is `resolveBundleFilePath(baseDir, resolveBundleFilePath(baseDir, f))`?

Reply:
It should be idempotent (from testing it), but removed for readability.